pyproject.toml

```diff
@@ -52,7 +52,7 @@ deprecation = "^2.0"
 jinja2 = "^3.0"
 jsonschema = "~3.2"
 networkx = "~2.6"
-packaging = ">=16.8,<22.0"
+packaging = ">=16.8"
```

Comment:
Allowing major version update is a little risky.  i think we should pin to a major version.  looks like version `24.1` works now so how about setting to "^24.1" instead?